manifests/role/training.pp

```diff
@@ -1,20 +1,23 @@
 class bootstrap::role::training inherits bootstrap::params {
   include bootstrap
+  include userprefs::defaults
   include bootstrap::profile::ruby
   include bootstrap::profile::rubygems
   include bootstrap::profile::cache_modules
   include bootstrap::profile::cache_rpms
   include bootstrap::profile::cache_docker
   include bootstrap::profile::pdf_stack
   include bootstrap::profile::network
-  include userprefs::defaults
   include bootstrap::profile::splash
   include bootstrap::profile::cache_wordpress
   include bootstrap::profile::cache_gems
-  class { 'bootstrap::public_key': 
+  include bootstrap::profile::disable_selinux
+
+  class { 'bootstrap::public_key':
     ec2_lock_passwd => false,
+    additional_key  => $bootstrap::params::admin_ssh_key
```

Comment:
@binford2k Should the parameter value be `$bootstrap::params::additional_key` here?